addOns/spider/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Include anti-csrf tokens as part of irrelevant parameters.
 - Ignore irrelevant parameters in request bodies (`x-www-form-urlencoded`) (Related to Issue 7771).
+- Skip all `javascript` schemes.
```

Comment:
Skip all &lt; &gt; `javascript` schemes.

Where &lt; &gt; is "links", "URLs", or "destinations".

Reply:
Which other `javascript` schemes do you know of?

Reply:
Sorry I should have said `Skip all <> with javascript schemes.`

Reply:
That does not answer the question, but I can be more explicit just in case.

Reply:
Okay fair point, I was just trying to make the entry a bit more user friendly. Not that it's wrong, just that it could be more obvious or straight forward.